./build.sh can't find clang or clang++ after fresh install

First-timer here. Ubuntu 16.04. Installing just `clang-3.8` doesn't set up a symlink for `clang`, which `build.sh` expects.

There appear to be two canonical ways to solve this: [using update-alternatives](https://askubuntu.com/a/700362/259428) and [installing the clang package](https://askubuntu.com/a/626178/259428).

I first solved my problem in two ways: by adding 
```
set(CMAKE_C_COMPILER "/usr/bin/clang-3.8")
set(CMAKE_CXX_COMPILER "/usr/bin/clang-3.8")
set(CMAKE_ASM_COMPILER "/usr/bin/clang-3.8")
```
to the top of `CMakeLists.txt` (doesn't seem very portable since it's a specific clang version) and by symlinking `/usr/bin/clang-3.8` to `/usr/bin/clang` (plus same for `clang++`).

Submitting a PR now to add `clang` to the dependency list in the README.

---

Here's the error message when running ./build.sh:
```
root@667b3968f7af:/cb-multios# ./build.sh
Creating build directory
Creating Makefiles
-- The C compiler identification is unknown
-- The CXX compiler identification is unknown
CMake Error at CMakeLists.txt:4 (project):
  The CMAKE_C_COMPILER:

    clang

  is not a full path and was not found in the PATH.

  Tell CMake where to find the compiler by setting either the environment
  variable "CC" or the CMake cache entry CMAKE_C_COMPILER to the full path to
  the compiler, or to the compiler name if it is in the PATH.


CMake Error at CMakeLists.txt:4 (project):
  The CMAKE_CXX_COMPILER:

    clang++

  is not a full path and was not found in the PATH.

  Tell CMake where to find the compiler by setting either the environment
  variable "CXX" or the CMake cache entry CMAKE_CXX_COMPILER to the full path
  to the compiler, or to the compiler name if it is in the PATH.


-- Configuring incomplete, errors occurred!
See also "/cb-multios/build/CMakeFiles/CMakeOutput.log".
See also "/cb-multios/build/CMakeFiles/CMakeError.log".
root@667b3968f7af:/cb-multios#
```

--- a/README.md
+++ b/README.md
@@ -42,7 +42,7 @@ libc6-dev libc6-dev-i386 gcc-multilib g++-multilib clang cmake
 ```
 
 ```
-sudo pip install xlsxwriter pycrypto defusedxml
+sudo pip install xlsxwriter pycrypto defusedxml pyyaml matplotlib
 ```
 
 To build all challenges, run:

--- a/README.md
+++ b/README.md
@@ -38,11 +38,11 @@ The following steps will build both the patched and unpatched binaries in `build
 The following packages are required for building the challenges on Linux:
 
 ```
-libc6-dev
-libc6-dev-i386
-gcc-multilib
-g++-multilib
-clang-3.8 (or higher)
+libc6-dev libc6-dev-i386 gcc-multilib g++-multilib clang cmake
+```
+
+```
+sudo pip install xlsxwriter pycrypto
 ```
 
 To build all challenges, run:

--- a/README.md
+++ b/README.md
@@ -42,7 +42,7 @@ libc6-dev libc6-dev-i386 gcc-multilib g++-multilib clang cmake
 ```
 
 ```
-sudo pip install xlsxwriter pycrypto
+sudo pip install xlsxwriter pycrypto defusedxml
 ```
 
 To build all challenges, run:

